src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10700,6 +11024,82 @@ namespace OutputReportTabular {
         }
     }
 
+    void WriteHeatEmissionTable()
+    {
+
+        // SUBROUTINE INFORMATION:
+        //       AUTHOR         Xuan Luo
+        //       DATE WRITTEN   November 2018
+        //       MODIFIED       na
```

Comment:
Remove all boilerplate comments, just simple comments to describe what the function does.
@JasonGlazer Would this have been easier to set up using the predefined report methods? Is that the preferred method for new reports? Not saying this needs to be changed now, just wondering.

Reply:
@mjwitte Thanks Mike! For functions in this namespace I'm doing a slight change of design for better aggregating the monthly report variables. Haven't push it yet as unit tests need to be modified, and my clang formatting config in Visual Studio is somehow messed up at this time point... Will do soon and let reviewers know.

But I have been switching back and forth between predefined or written tables, so it would be great if I can get some general advice at this point.